tests/Unit/ControlSystem/Test_Measurement.cpp

```diff
@@ -0,0 +1,112 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <optional>
+#include <type_traits>
+
+#include "ControlSystem/Event.hpp"
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Evolution/EventsAndDenseTriggers/EventsAndDenseTriggers.hpp"
+#include "Framework/ActionTesting.hpp"
+#include "Helpers/ControlSystem/Examples.hpp"
+#include "Options/Protocols/FactoryCreation.hpp"
+#include "Parallel/PhaseDependentActionList.hpp"
+#include "Parallel/Tags/Metavariables.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Event.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+using MeasureEvent = control_system::Event<
+    tmpl::list<control_system::TestHelpers::ExampleControlSystem>>;
+
+template <typename Metavariables>
+struct ElementComponent {
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockArrayChare;
+  using array_index = int;
+  using phase_dependent_action_list = tmpl::list<
+      Parallel::PhaseActions<typename Metavariables::Phase,
+                             Metavariables::Phase::Testing, tmpl::list<>>>;
+};
+
+template <typename Metavariables>
+struct MockControlSystemComponent {
+  using component_being_mocked = control_system::TestHelpers::
+      ReplaceThisWithControlSystemComponentWhenThatIsWritten<
+          control_system::TestHelpers::ExampleControlSystem>;
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockSingletonChare;
+  using array_index = int;
+  using initialization_tags = tmpl::list<
+      control_system::TestHelpers::ExampleControlSystem::MeasurementQueue,
+      Tags::Time, control_system::TestHelpers::MeasurementResultTag>;
```

Comment:
For a test it probably doesn't matter, but I generally haven't needed to add the time to the control component databox. Also, I don't think MeasurementQueue is strictly needed for initialization (only needed in simple_tags), but I guess since you aren't running the SetupDataBox initialization action this is probably the best way to get tags into the databox.

Reply:
The time is used to check the test output, but it would probably be better to use a custom tag instead of the standard Time tag to make it clear that it isn't being used for anything else.

There are at least two seemingly equivalent methods of initializing DataBoxes in the tests, and I don't understand why or which one is preferred.